regression/python/string-replace_fail/main.py

```
@@ -0,0 +1,6 @@
+def main() -> None:
```

Can you add a test case with symbolic strings via `nondet_str()`? Even if we don't handle it yet, we should raise an error message via `log_error()` to inform the user that we don't support it yet.